library.gradle

```diff
@@ -41,4 +62,4 @@ dependencies {
     testImplementation 'org.assertj:assertj-core:3.13.2'
 }
 
-apply plugin: 'com.vanniktech.maven.publish'
+//apply plugin: 'com.vanniktech.maven.publish'
```

Comment:
I wasn't able to get the maven plugin to work with the new flavors... I removed it for now